### PR TITLE
{bio}[intel/2019b] MultiQC v1.8 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3402.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/710d501c1847e80fb9d874ab7f1b4d15 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0a5401028dac67f179850e597ded7deb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2452.golett.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/7e9d51583d07de975a74a69407aa1230 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3128.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/508cdf31e7c97d2a2d02cb0ac06a63b9 for a full test report.

File: easybuild/easyconfigs/m/MultiQC/MultiQC-1.8-intel-2019b-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# Note that Click in Python 3 requires that you change your locale to unicode before invoking your Python script.
+# See: https://click.palletsprojects.com/en/7.x/python3/
+
+easyblock = 'PythonBundle'
+
+name = 'MultiQC'
+version = '1.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://multiqc.info'
```
Comment (Micket) [inline easybuild/easyconfigs/m/MultiQC/MultiQC-1.8-intel-2019b-Python-2.7.16.eb:17]:
```suggestion
homepage = 'https://multiqc.info'
```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/95d34b4098001f36b21f27abab7f7c29 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**